boskos/gcp/config_test.go

```diff
@@ -51,6 +51,7 @@ func TestParseInvalidConfig(t *testing.T) {
 					},
 					Scopes: []string{
 						"https://www.googleapis.com/auth/cloud-platform",
+						"https://www.googleapis.com/auth/trace.append",
```

Comment:
Don't you need to add this under cluster ?

Reply:
You are right! Good catch